components/Nav.js

```diff
@@ -34,9 +34,7 @@ export default ({ file, versions, dispatch }) => {
 
   const handleVersionChange = v => {
     changeVersion(v);
-    const [, path] = file.url.match(
-      /https:\/\/unpkg.com\/(?:@?[^@\n]*)@?(?:\d+\.\d+\.\d+)(.*)$/
-    );
+    const [, path] = file.url.match(/https:\/\/unpkg.com\/[^\/]*(\/?.*)$/);
```

Comment:
It doesn't actually break it, but I realised we should strictly escape the dot in `.com` (i.e. ...`\/\/unpkg\.com`...)

Reply:
Good eye! I pushed the suggestion. :smile: